### PR TITLE
[IMP] sales: invoicing policy doc for RST reasons

### PR DESCRIPTION
**Pull request by ksc-odoo**
updated sales doc for invoicing policies for RST reasons and image reasons, suggested in this task https://www.odoo.com/web#id=3324366&cids=3&model=project.task&view_type=form
This is just for 16.0, as the images are version-specific

Project Task: https://www.odoo.com/web#id=3330511&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:12]:
```suggestion
is often used for businesses that sell materials, liquids, or food in large quantities. In these cases,
the ordered quantity may differ from the delivered quantity, making it preferable to invoice the quantity
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:32]: I think this rewrite improves the clarity. I also removed an instance of "you".
```suggestion
   If the :guilabel:`Invoice what is delivered` rule is chosen, it is **not** possible to activate the :guilabel:`Automatic
   invoice` feature, which automatically generates invoices when an online
   payment is confirmed.
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:39]:
```suggestion
On any product page (:menuselection:`Sales app --> Products --> Products --> Choose any desired
product), the invoicing policy option is located under the :guilabel:`General Information` tab, where
it can be changed manually.
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:60]:
```suggestion
  needs to be manually entered on the sales order. Or, the *Inventory* app can be installed and
  used to confirm the delivered quantity before creating an invoice with the *Sales* app. If a user attempts to create an invoice without validating the delivered quantity, the following
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:69]:
```suggestion
   Once a quotation is confirmed and changes to a Sales Order, the delivered and invoiced quantities are available to view directly
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:78]:
```suggestion
Finally, there are two options for creating an invoice: *Regular invoice* or
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:83]: You could also just hyperlink the first mention of "down payment".
```suggestion
   Be sure to check out the documentation explaining down payment options to learn more:
   :doc:`/applications/sales/sales/invoicing/down_payment`.
```

**Review by hojo-odoo — COMMENTED** [review]
Hey @ksc-odoo,

Let me know if you have any questions about my comments.

Thanks,
hojo

**Comment by ksc-odoo** [comment]
Thanks for the super-helpful feedback/suggestions, @hojo-odoo - I made all the necessary adjustments, and I think it's ready for a Tech Review from @samueljlieber - Thanks again 👍

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:11]: Early line break
```suggestion
- The *Invoice what is delivered* rule invoices customers once the delivery is done. This rule is
  often used for businesses that sell materials, liquids, or food in large quantities. In these
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:23]: Early line break
```suggestion
:guilabel:`Invoicing Policy` rule: :guilabel:`Invoice what is ordered` or :guilabel:`Invoice what is
delivered`.
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:27]: Corrected alt text
```suggestion
   :alt: Choosing an invoicing policy on Odoo Sales.
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:39]: Adding gui since were referring to the field in the instruction
```suggestion
product), the :guilabel:`Invoicing Policy` option is located under the :guilabel:`General
Information` tab, where it can be changed manually.
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:65]: This text is relying too much on the image for context, and the alt text for the image does not explain the text in the image.

To resolve this, add the text in the image somewhere to the bullet point OR use an admonition to highlight the warning– with the admonition the image may not even be necessary 🙂
```suggestion
**- :guilabel:`Invoice what is delivered`: Minor impact on sales flow, because the delivered quantity
  needs to be manually entered on the sales order. Or, the *Inventory* app can be installed and used
  to confirm the delivered quantity before creating an invoice with the *Sales* app.

  .. warning::
     If a user attempts to create an invoice without validating the delivered quantity, the
     following error message appears: "There is no invoiceable line. If a product has a Delivered
     quantities invoicing policy, please make sure that a quantity has been delivered".

     .. image:: invoicing_policy/invoicing-policy-error-message.png
        :align: center
        :alt: There is no invoiceable line. If a product has a Delivered quantities invoicing
              policy, please make sure that a quantity has been delivered.**
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:79]: I feel these should be written as they are in the UI with guilabels, but I think the way you wrote it would also work just with lowercase in the italics
```suggestion
Finally, there are a few different options to create an invoice: :guilabel:`Regular invoice`,
:guilabel:`Down payment (percentage)` or :guilabel:`Down payment (fixed amount)`.
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:83]: Remove redundant words
```suggestion
   Be sure to check out the documentation explaining down payment options to learn more:
   :doc:`/applications/sales/sales/invoicing/down_payment`.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! This PR is looking pretty good :) I have just a couple technical corrections, please see below and then tag me again for another look!

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo! This PR is looking pretty good :) I have just a couple technical corrections, please see below and then tag me again for another look!

@samueljlieber just made all your suggested edits - ready for another review from you, whenever you get a chance 👍 Thanks!

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:9]: Can remove extra line– no need for an extra line between bullet points of the same level :)
```suggestion
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:56]: Remove extra line
```suggestion
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:64]: This is text that appears in the UI, so it deserves a guilabel. I think you could include the double quotes if you wanted too 🙂
```suggestion
   error message appears: :guilabel:`There is no invoiceable line. If a product has a Delivered
   quantities invoicing policy, please make sure that a quantity has been delivered.`.
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo 👋 great job with the improvements to this doc! This PR is looking great to me, I found just a few more small improvements. Please see below and correct them before sending to final review. I am approving now 🙂

**Comment by ksc-odoo** [comment]
@StraubCreative this one's ready for Final Review 👍

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (meng-odoo) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:77]: It's unclear which "quantities" this sentence is referring to. The delivered or invoiced quantities? When does this happen?

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Great job on this doc. I just had tiny formatting changes and one clarifying question. Once you've made those changes, we'll tag Sam for one more quick technical review before merging. Please let me know if you have any questions :)

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo! Great job on this doc. I just had tiny formatting changes and one clarifying question. Once you've made those changes, we'll tag Sam for one more quick technical review before merging. Please let me know if you have any questions :)

Hey there, @meng-odoo -- just finished making all your suggested edits. Thanks for the thorough review. I believe this is now ready for another quick technical review from @samueljlieber 👍

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:66]: Small change– indent image by one more space :)
```suggestion
   .. image:: invoicing_policy/invoicing-policy-error-message.png
      :align: center
      :alt: If Delivered Quantities invoicing policy is chosen, ensure a quantity has been
            delivered.
```

**Review by samueljlieber — APPROVED** [review]
Hey @ksc-odoo 👋 great job with the improvements in this PR, I found a single very small technical change. I am approving now, but please make this correction before tagging for merge. Nice work!

**Comment by ksc-odoo** [comment]
> Hey @ksc-odoo 👋 great job with the improvements in this PR, I found a single very small technical change. I am approving now, but please make this correction before tagging for merge. Nice work!

Hey @samueljlieber - thanks for the quick review. Made that suggested edit, so I believe this is ready to merge... ready for your final review @StraubCreative

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:37]: Creative use of menuselection 😄  
Here's my suggestion to fit a more typical use case for this tag.

```suggestion
On any product page via the :menuselection:`Sales app --> Products --> Products dashboard, locate the :guilabel:`Invoicing Policy` option located under the :guilabel:`General
```

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:24]: can expand the view here to show other UI elements.

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:40]: prefer red rectangles in most cases, since arrows add visual noise to the screenshot

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:83]: seealso tag

File: content/applications/sales/sales/invoicing/invoicing_policy.rst
Comment (StraubCreative) [inline content/applications/sales/sales/invoicing/invoicing_policy.rst:81]: more info here or seealso links to relevant documentation?

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 
Approving with comments for you to consider now or on a future PR.
At some point, we'll want to update the screenshots to feature the UI, as well.

Up to you, however we can merge right away if you like 🙏 

@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
